spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1352,6 +1372,7 @@ def request_folding(self):
     @handles(LSPRequestTypes.DOCUMENT_FOLDING_RANGE)
     def handle_folding_range(self, response):
         """Handle folding response."""
+        self.request_symbols()

```

Comment:
@ccordoba12 tests were failing if I moved the symbol request where you told me so I put it here after the folding request. Not sure what the problem was 🤷‍♂

Reply:
I pushed a commit here to better address this.